poetry.lock

```diff
@@ -59,7 +59,7 @@ cchardet = "*"
 
 [[package]]
 name = "alembic"
-version = "1.6.2"
+version = "1.6.5"
```

Comment:
Generally speaking how do we manage these .lock files?

Reply:
In this case it is inconsequential since I added alembic but have not yet set it up (it provided database schema migration across versions). Ideally there should be test coverage to verify that an update has no negative impact. In reality, we see if things still work and continue on.

Reply:
Yeah, if you view the full diff you'll see there are many many lines that changed.  Is the general workflow to run `poetry update` then let it update this file then check it in?

Maybe as part of this exercise (or shortly after the work is complete) I will write a test client that exercises the API as it is known to exist today and makes assertions against known good data.